addons/spreadsheet/static/src/global_filters/components/filter_date_from_to_value/filter_date_from_to_value.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <div t-name="spreadsheet_edition.DateFromToValue" class="date_filter_values">
+        <DateTimeInput
+            value="from"
+            type="'date'"
+            placeholder="'Date from...'"
```

Comment:
strings defined like this aren't translated. Move this string to be a component attribute, translated with _t